README.md

```diff
@@ -24,173 +24,230 @@ Docker images are the preferred way to deploy this code in a production environm
 
-```shell
-make run
+```sh
+curl http://localhost:6500/
 ```
 
-When you need to stop the application and containers and delete everything to start fresh, first check the path of the
-postgres docker container volume in `docker-compose-dev.yml`. Update the `POSTGRES_DATA` in the Makefile to match it
-and then run:
+Check out the [Docker README](/docker/README.md) in the `/docker` directory for in-depth information on running and
+developing the Circulation Manager locally with Docker, or for deploying the Circulation Manager with Docker.
 
-```shell
-make clean
-```
+For setting up a library with collections, you can skip over to section [The Admin Interface](#admin-interface).
 
-You can modify the Makefile to also create a virtual enviroment for Python 3.10.1. You can switch between the virtual
-environments by activating and deactivating them with e.g. `pyenv activate circ-311` and `pyenv deactivate circ-311`.
+### 2. Python and dependencies to run tests (and the application locally)
 
-### Docker Compose
+#### Python Setup
 
-In order to help quickly set up a development environment, we include a [docker-compose.yml](./docker-compose.yml)
-file. This docker-compose file, will build the webapp and scripts containers from your local repository, and start
-those containers as well as all the necessary service containers.
+In order to run tests or the code outside Docker, you'll need to set up Python and a virtual environment. Macs should
+have Python already installed.
 
-You can give this a try by running the following command:
+#### Dependencies
 
-```shell
-docker-compose up --build
+Install the following required dependencies:
+
+```sh
+brew install pkg-config libffi
+brew install tvuotila/libxmlsec1/libxmlsec1@1.2.37
+brew install libjpeg
 ```
 
-### Python Set Up
+#### venv - Virtual environment
 
-#### Homebrew (OSX)
+The codes uses Python 3.10 and 3.11. We mostly use 3.11, but install both versions (3.10 in case you want to run
+tests against it):
 
-If you do not have Python 3 installed, you can use [Homebrew](https://brew.sh/) to install it by running the command
-`brew install python3`.
+```sh
+brew install python@3.11
+brew install python@3.10
+```
 
-If you do not yet have Homebrew, you can install it by running the following:
+Create a virtual environment that uses Python 3.11 and activate it:
 
 ```sh
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+python3.11 -m venv venv
+source venv/bin/activate
 ```
 
-While you're at it, go ahead and install the following required dependencies:
+#### Poetry
+
+This project uses [poetry](https://python-poetry.org/) for dependency management.
+
+Poetry can be installed using the command `curl -sSL https://install.python-poetry.org | python3 -` but at the moment,
+Poetry version 1.8.3 works without problems. Install it:
 
 ```sh
-brew install pkg-config libffi
-brew install libxmlsec1
-brew install libjpeg
+brew install poetry@1.8.3
```

Comment:
Could not be installed via Homebrew. Instead, I installed first pipx via Homebrew, then `pipx install poetry==1.8.3`. I also had to add Poetry directory to PATH environment.